### ISSUE TITLE
Chat with friends

### ISSUE THREAD

**Issue by klausenbusk** [desura/Desurium#156]
It could be so cool, if we could chat with friends :)

**Comment by LiamDawe** [comment]
This has been requested before.

**Comment by klausenbusk** [comment]
Maybe it just me, but i cant find a issues for it?

**Comment by Protektor-Desura** [comment]
I am all in favor of this and want to use XMPP and Jingle. XMPP gives us all the chat stuff we need and it is compatible with other networks. Origin, Facebook and Google are currently using XMPP for their chat systems. I haven't looked at Steam in awhile but last I looked they didn't offer chat just sending PMs. 

I also want to use Jingle extensions for XMPP from Google to add voice chat, video chat (maybe) and file sending/receiving. These would be great features for development teams especially MOD development teams.

Google has released a set of cross-platform libraries for all of this under the BSD license. So adding it to the client would not be a license conflict and would mean we don't have to develop it ourselves.

**Comment by LiamDawe** [comment]
Steam actually has full chat support with friends lists and game invites, including chat rooms btw just for reference.

I think maybe this could be a focus of initial release + 1?

**Comment by Gigadoc2** [comment]
I think we're all in favor of this, and we all want to use XMPP.
But for this feature, either support from Desura Ltd. is needed (no XMPP without a server), or we have to tell the users to register at third-party servers.

**Comment by ssokolow** [comment]
It'd be nice to have both. A Desura-provided XMPP account for effortless setup and support for 3rd-party ones for people who would rather do fancier things with their communications setup.

**Comment by WhyNotHugo** [comment]
Federated XMPP support would be great. It wouldn't take much effort on desura's behalf to set up a prosody server using existing desura accounts. Having server-side support should come before any client-side work can get done.
Does anyone have any idea what the intetion is on desura's behalf?
(note: By "desura" I mean "Desura Ltd.")

**Comment by karolherbst** [comment]
nah, we could create a GUI widget for XMPP support which is connecting anywhere else for testing. Some work should be done after it, too. For example: link to member profile, current playing game, etc...

**Comment by karolherbst** [comment]
I will ask about the plans

**Comment by karolherbst** [comment]
mhh, We could implement some XMPP stub stuff now, but we have to wait for the server support a bit.
